server/src/main/java/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -1946,12 +1946,6 @@ private boolean upgradeRunningVirtualMachine(Long vmId, Long newServiceOfferingI
                     _itMgr.reConfigureVm(vmInstance.getUuid(), currentServiceOffering, newServiceOffering, customParameters, existingHostHasCapacity);
                     success = true;
                     return success;
-                } catch (InsufficientCapacityException e) {
-                    s_logger.warn("Received exception while scaling ", e);
-                } catch (ResourceUnavailableException e) {
-                    s_logger.warn("Received exception while scaling ", e);
-                } catch (ConcurrentOperationException e) {
-                    s_logger.warn("Received exception while scaling ", e);
                 } catch (Exception e) {
                     s_logger.warn("Received exception while scaling ", e);
```

Comment:
@Pearl1594 do we need to catch all exceptions here?

Reply:
@Pearl1594 sorry for the confusion but I was referring `catch (Exception e)`. Do we really need that?
Anyway, not your change so I'm LGTM for this PR